database/test_data.py

```diff
@@ -864,6 +867,56 @@
 
 EmailTemplate.insert_many(emailTemplates).on_conflict_replace().execute()
 
+emailLogs = [
+    {
+    'event': 5,
+    'subject': 'Location Change for {event_name}',
+    'templateUsed': 2,
+    'recipientsCategory': "RSVP'd",
+    'recipients': 'neillz',
+    'dateSent': datetime.strptime("2022 5 7","%Y %m %d"),
+    'sender': User.get_by_id("neillz")
```

Comment:
In the test data don't retrieve the user object, just use the id. In this case, `neillz`